### DISCUSSION TITLE
pdf双语翻译中文可复制

### DISCUSSION THREAD

**Discussion by thomasTAT** [immersive-translate/immersive-translate#1631] [Ideas]
### 新功能 | New Feature

pdf双语翻译中文可复制 

### 为什么您想要添加这些功能 | Why you want to add these features

有时候想复制翻译结果

### 提议 | Proposal

OCR

**Comment by keyboard3** [comment]
没懂，在线双语对照不是可以复制吗？

**Reply by thomasTAT** [reply]
pdf 双语翻译后，导出后译文不能复制。

**Reply by keyboard3** [reply]
导出双语是图片格式，无法复制。可以选择仅译文打印模式或者快照功能分享出仅译文
